### PR TITLE
[nextcloud] Containerized nextcloud build script

### PR DESCRIPTION
**Pull request by pabloyoyoista**
Still missing:
- Update name of registry in FROM directive
- Review README file

### CODE REVIEW

File: nextcloud/Containerfile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	    php-imagick; \
+    mkdir /var/www ; \
+    wget -qO- https://download.nextcloud.com/server/releases/latest-${VERSION}.tar.bz2 | \
+    tar -C /var/www -xjv --no-same-owner; \
+    chown www-data /var/www/nextcloud/apps; \
+    chown www-data /var/www/nextcloud/config
+
+RUN DIR="/var/www/nextcloud/apps/richdocuments"; \
+    git clone https://github.com/nextcloud/richdocuments.git $DIR ; \
+    git --git-dir=$DIR/.git --work-tree=$DIR checkout $(git --git-dir=$DIR/.git --work-tree=$DIR tag | tail -n1)
```
Comment (Akasch) [inline nextcloud/Containerfile]: clever, but I am not sure the ordering is correct (current versions have a "v" prefix in the version number some older releases not)

**Review by Akasch — CHANGES_REQUESTED** [review]
seems good apart from comment, maybe look into the offical image how they install plugins?

File: nextcloud/Containerfile [hunk 2]
```diff
@@ -0,0 +1,53 @@
+FROM dir://base-buster
```
Comment (timonegk) [inline nextcloud/Containerfile]:
```suggestion
FROM registry.mafiasi.de/base-buster
```
would probably be better

**Review by timonegk — CHANGES_REQUESTED** [review]

File: nextcloud/Containerfile (hunk 2 above)
Comment (pabloyoyoista) [reply nextcloud/Containerfile]: Fixed

File: nextcloud/Containerfile (hunk 1 above)
Comment (pabloyoyoista) [reply nextcloud/Containerfile]: Now it should in principle work.

However, I wonder whether we should automatically select the last tag or set a constant version. At least until the CI is up and working.

Other than that, the official images don't install any extra plugins.

**Comment by lilioid** [comment]
nextcloud is now already converted although not via pull request

**Pull request closed without merging**